ClassicPress-research/classic-commerce
# Change status report for Classic Commerce

### All Submissions:


### Changes proposed in this Pull Request:

Change status report to reference Classic Commerce where necessary. Change references to WordPress to make the report more generic, so that it is applicable to either ClassicPress or WordPress. Simplify some areas to remove conditional testing and reporting of results, as these often include references to the WooCoomerce or WordPress docs. In general, recommendations for settings have been moved to the help hover message, making it easier to maintain in the long term.

### How to test the changes in this Pull Request:

1. Copy the revised file html-admin-page-status-report.php to a working Classic Commerce installation.
2. Run the report and check the results are accurate.
3. Disable and enable various features and re-run the report to make sure they are correctly showing up in the new report.

### Other information:


### Changelog entry

Edit status report to reference Classic Commerce where necessary. Remove specific references to WordPress to make the report more generic.
